opentargets.json

```diff
@@ -1323,11 +1323,13 @@
     "studyStartDate": {
       "type": "string",
       "format": "date",
-      "description": "Start date of study in a YYYY-MM-DD format"
```

Comment:
Shouldn't we just request data field? [JSON Schema allows this type](http://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1). I'm not 100% sure it applies here though.